components/Footer.js

```diff
@@ -3,9 +3,8 @@ import Link from "next/link";
 export default function Footer() {
   return (
     <footer className="flex justify-center mt-8 italic">
-      Powered by{" "}
       <Link href="https://github.com/EddieHubCommunity/LinkFree">
-        <span>EddieHub</span>
+        Powered by EddieHub
```

Comment:
Would this make it all an link? 🤔

Reply:
I think only EddieHub should be linked, because if everything is formatted the same way, nothing is standing out?